index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio: setup & mobile first</title>
+    <!-- link to style.css -->
+    <link rel="stylesheet" href="style.css">
+    <!-- link to roboto & poppins & RoundFamily & Inter fonts: copied from google fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins:wght@300&family=Roboto&display=swap" rel="stylesheet">
+    <!-- link to my burger icon -->
+    <script src="https://kit.fontawesome.com/8f4917926e.js" crossorigin="anonymous"></script>
+</head>
+
+
+<body>            
+    <!-- my header with a navegation menu. Contains my logo and my "burger" menu button -->
+    <header class="header">
+        <nav class="nav">
+        <div class="my-logo"><h1 >My logo</h1></div>    
+            <!-- import my burger menu from Font Awsome-->
+        <div class="burger-menu"><i class="fa-solid fa-bars"></i></div>    
+        </nav>
+
+         <!-- main content on my web -->  
+        <main class="main">
+            <div>
+            <h2 class="h-1">Hi there.<br>I'm Nico.</h2>
+            <h3 class="h-2">I'm a Sofware Developer</h3>
```

Comment:
- [ ] Good work developing your headline title and subtitle!✔️ However, please kindly make sure that you wrap "Hi there, I'm Nico" with an h1 tag since this is the principal header of your headline section. 🤓 Also, please kindly change the h3 tag from your subtitle to an h2 tag for the same reason. 🔧